phaseshifts/leed.py

```
@@ -445,43 +422,31 @@ def import_CLEED(filename, **kwargs):
-        ]
+        basis = [line for line in lines if line.startswith("a") and int(line[1]) <= 3 and int(line[1]) > 0]
 
         a1 = a2 = a3 = [0.0, 0.0, 0.0]  # intialise basis vectors
         for vector_line in basis:
             line = " ".join(vector_line.replace(":", "").split()[:4])
 
             try:
-                (vector_str, x, y, z) = (
-                    t(s) for (t, s) in zip((str, float, float, float), line.split())
-                )
+                (vector_str, x, y, z) = (t(s) for (t, s) in zip((str, float, float, float), line.split()))
```

Ruff (0.14.10)

`zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)